web/src/components/RunDetailTest/RunDetailTest.tsx

```diff
@@ -77,8 +151,69 @@ const RunDetailTest = ({run, testId}: IProps) => {
                     setPrevSelector(prevSelector);
                   }}
                 />
-              ) : (
-                <TestResults />
+              )}
+
+              {isTestOutputFormOpen && (
+                <TestOutputForm
+                  isEditing={isEditing}
+                  isLoading={isLoading}
+                  onCancel={onClose}
+                  onSubmit={onSubmitTestOutput}
+                  output={output}
+                  runId={run.id}
+                  testId={testId}
+                />
+              )}
+
+              {!isTestSpecFormOpen && !isTestOutputFormOpen && (
```

Comment:
It feels like we are trying to jiggle the two concepts and if feels weird haha.

Not for this PR but maybe we can consider consolidating a single draft mode where multiple things can be added to the bucket

Reply:
Like the idea of one single draft mode.